examples/mobile/src/lib.rs

```diff
@@ -161,3 +165,18 @@ fn setup_music(asset_server: Res<AssetServer>, mut commands: Commands) {
         settings: PlaybackSettings::LOOP,
     });
 }
+
+// Pause audio when app goes into background and resume when it returns.
+// This is handled by the OS on iOS, but not on Android.
+fn handle_lifetime(
+    mut lifetime_events: EventReader<Lifetime>,
+    music_controller: Query<&AudioSink>,
+) {
+    for event in lifetime_events.read() {
+        match event {
+            Lifetime::Suspended => music_controller.single().pause(),
+            Lifetime::Resumed => music_controller.single().play(),
```

Comment:
Isn't this going to play any existing AudioSink, regardless of whether they were paused or not before the suspension? 😅